examples/undocumented/python/graphical/mclda.py

```diff
@@ -1,63 +1,57 @@
-from shogun import RealFeatures
 from shogun import MulticlassLabels
 from shogun import MCLDA
-from pylab import pcolor, contour, colorbar, connect, show, plot, axis
-
+import matplotlib.pyplot as plt
 import numpy as np
-import util
+from shogun import features
 
 N = 500
 size = 100
 
-
 # positive examples
 mean_pos = [-1, 4]
-cov_pos  = [[1,40], [50, -2]]
+cov_pos = [[1, 40], [50, -2]]
 
-x_pos, y_pos = np.random.multivariate_normal(mean_pos, cov_pos, 500).T
-plot(x_pos, y_pos, 'bo');
+x_pos, y_pos = np.random.multivariate_normal(mean_pos, cov_pos, N).T
+plt.plot(x_pos, y_pos, 'bo')
 
 # negative examples
 mean_neg = [0, -3]
-cov_neg  = [[100,50], [20, 3]]
+cov_neg = [[100, 50], [20, 3]]
 
-x_neg, y_neg = np.random.multivariate_normal(mean_neg, cov_neg, 500).T
-plot(x_neg, y_neg, 'ro');
+x_neg, y_neg = np.random.multivariate_normal(mean_neg, cov_neg, N).T
+plt.plot(x_neg, y_neg, 'ro')
 
 # train qda
-labels = MulticlassLabels( np.concatenate([np.zeros(N), np.ones(N)]) )
+labels = MulticlassLabels(np.concatenate([np.zeros(N), np.ones(N)]))
 pos = np.array([x_pos, y_pos])
 neg = np.array([x_neg, y_neg])
-features = RealFeatures( np.array(np.concatenate([pos, neg], 1)) )
+
+features_ = features(np.array(np.concatenate([pos, neg], 1)))
```

Comment:
@karlnapf maybe we should stop `from shogun import *` to avoid this kind of thing happening?

Reply:
yes absolutely, we should use `import shogun as sg` and then `sg.features` (actually also in meta examples, Ill open an issue)

Reply:
#4990